src/applications/mhv-medical-records/components/DownloadRecords/DownloadRecordType.jsx

```diff
@@ -76,6 +79,22 @@ const DownloadRecordType = () => {
     setSelectedRecords(newArray);
   };
 
+  useEffect(() => {
+    if (!dateFilter) {
+      history.push('/download/date-range');
+    }
+  });
```

Comment:
useEffects without dependency arrays are unpredictable. I would think you would want `[dateFilter]` here.